api/tests/test_api.py

```diff
@@ -0,0 +1,41 @@
+#test_api.py
+import unittest, json
+
+from api import app 
+
+#class to respresent books testcase
+class TestCreateBooks(unittest.TestCase):
```

Comment:
rename the class `TestBooks` you will have a function `test_add_books` and so on.